src/js/api/auth/login.js

```diff
@@ -77,3 +43,62 @@ function getRedirectUrl(role) {
       return '/pages/user/index.html';
   }
 }
+
+
+/**
+ * Function for logging in an existing user in database and storing the returned token in session or localstorage
+ * @param {object} profile Values from loginForm
+ * @param {string} profile.email Email of the user
+ * @param {string} profile.password Plain text password
+ * @param {string} [profile.remember] If the user checkbox is checked it will equal to the string 'on'
+ * @returns {void}
+ */
+export async function login(profile) {
+  const { remember, email, password } = profile;
+
+  const rememberLogin = remember === 'on';
+
+  const errorContainer = document.querySelector('#errorContainer');
+
+  try {
+    const { userData, error } = await callLoginApi(email, password);
+
+    if (error) {
+      return (errorContainer.innerHTML = error?.message);
+    }
+
+    const { role } = userData;
+
+
+        if (id === id) {
+            // spiderman.gif
+          new Store('Role', 'user', Boolean(remember !== 'on'));
+         window.location.replace('../../../pages/user/index.html');
+
+        } else if (profile.admin) {
+          new Store('Role', 'admin', Boolean(remember !== 'on'));
+          window.location.replace('#'); // TODO: Add admin page url
+        } else {
+          window.location.replace('../../../pages/user/index.html');
+        }
+        break;
+      case 403:
```

Comment:
There's no switch in this method, so I'm getting a syntax error. Can you fix it? 🙂

![Skjermbilde 2024-01-11 kl  17 16 29](https://github.com/NoroffFEU/agency.noroff.dev/assets/68667843/8f49c378-8ebc-474b-85ad-0c9f7a0b7b2c)

Reply:
Oh i just change the path of index.html. i didnt do anything i think.